torch/csrc/distributed/rpc/PythonRpcHandler.cpp

```diff
@@ -0,0 +1,89 @@
+#include <torch/csrc/distributed/rpc/PythonRpcHandler.h>
+
+namespace torch {
+namespace distributed {
+namespace rpc {
+
+PyObject* PythonRpcHandler::module_ = nullptr;
+PyObject* PythonRpcHandler::runUDFFunction_ = nullptr;
+PyObject* PythonRpcHandler::loadResultFunction_ = nullptr;
+
+void PythonRpcHandler::init() {
+  Py_Initialize();
```

Comment:
We could move this line, `Py_Initialize();`.

As documentation says,
> This initializes the table of loaded modules (sys.modules), and creates the fundamental modules builtins, __main__ and sys. It also initializes the module search path (sys.path). It does not set sys.argv;

Since the code has run to here, the interpreter system-level modules must have already been initialized, so no need to call initialize again.

Reply:
we still need this if the application is C++ application and no python environment. added to check Py_IsInitialized() or not first, to ensure Py_Initialize() is called only once